### ISSUE TITLE
Cant close "Activate Lingotek" popup message in 2.3.9

### ISSUE THREAD

**Issue by paulhuisman** [polylang/polylang#289]
Since updating to Polylang 2.3.9 there is a notification in the admin:

"You’ve just upgraded to the latest version of Polylang! Would you like to automatically translate your website for free? Click on Activate Lingotek to start translating."

When clicking the 'close' icon this does not dissapear.

Wordpress 4.9.8

**Comment by paulhuisman** [comment]
Using ` "wpackagist-plugin/polylang": "2.3.9"`  in composer.

**Comment by Chouby** [comment]
Yes the bug has already been reported in 2.3.9. It was fixed in the master branch. The 2.3.10 is currently beeing pushed.

**Comment by paulhuisman** [comment]
Ah okay. thanks!
